packages/react-components/react-card/MIGRATION.md

```diff
@@ -0,0 +1,322 @@
+- [From V0](#from-v0)
```

Comment:
I think it would be better to add the migration guide to the [public docsite](https://github.com/microsoft/fluentui/tree/master/apps/public-docsite-v9/src/Concepts/Upgrade/FromV0/Components) so that it appears on `react.fluentui.dev` and is more easily discoverable.

Reply:
I will also be added to the public site, but as a separate PR. The intent of this is to have a document on the package it self, as many other have it. I renamed this PR title to better reflect that. :)

Reply:
Also, the Card component is still unstable, so the API might change.